tools/docker/builder/rdkb/toolchain/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+echo RDKB_ROOT=$RDKB_ROOT
+cd $RDKB_ROOT/setup
+
+if [ -d "${RDKB_ROOT}/atom_rdkbos/build" ]; then 
+    . puma_setup -o 
+else
+    . puma_setup -c configs/p7_atom_rdkbos_dev-debug.config
```

Comment:
we use `configs/p7_atom_rdkbos_wave600_dev-debug.config`

Reply:
Thanks, I'll change it then. Good catch!